dev/main.sh

```diff
@@ -47,8 +47,8 @@ if [ "$existingTenants" == "[]" ]; then
     for tenant in "${tenants[@]}"
     do
         echo "🐈Provisioning tenant: $tenant"
-        baseURL="$tenant.stellar.local:8000"
-        sdpUIBaseURL="$tenant.stellar.local:3000"
+        baseURL="http://$tenant.stellar.local:8000"
+        sdpUIBaseURL="http://$tenant.stellar.local:3000"
```

Comment:
Since there are other ways to input this information (through APIs and CLI), we should actually change the code so our join logic can work for URLs that don't have the schema

Reply:
In my opinion, we should reject `BaserURL` and `SDPUIBaseURL` without the scheme/protocol because there's no way to determine which scheme we should use (`HTTP` or `HTTPS`). Relying on the standard library to join the URI is the best since it's battle-tested.

Reply:
I defer this to @marwen-abid